src/test/java/org/springframework/metrics/instrument/MetersTest.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.metrics.instrument;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.NavigableSet;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.offset;
+import static org.springframework.metrics.instrument.Tag.tags;
+
+/**
+ * @author Jon Schneider
+ */
+class MetersTest {
+    @Test
+    void customMeter() {
+        NavigableSet<String> letters = new TreeSet<>();
+        letters.addAll(Arrays.asList("a", "b", "c", "d", "e"));
+
+        Meter customMeter = Meters.build("letters_after")
+                .tags("letter", "b")
```

Comment:
@checketts Is it apparent enough that these tags get added to every measurement?